examples/sticky-ads-amp.html

```diff
@@ -0,0 +1,52 @@
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <title>Ad examples</title>
+  <link rel="canonical" href="http://nonblocking.io/" >
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <style amp-custom>
+    amp-sticky-ad {
```

Comment:
will this actually be the css for the element? if so, this should cs inside `amp-stick-ad.css`